docs/source/development/changes.md

```diff
@@ -1,5 +1,5 @@
 (changes)=
 # Changes
-```{eval-rst}
-.. include:: ../../../CHANGES.md
+
+```{include} ../../../CHANGES.md
```

Comment:
This makes the **Changes** header appear twice on the site.

Reply:
Done.